packages/xo-server-perf-alert/src/index.js

```diff
@@ -1,8 +1,11 @@
 import JSON5 from 'json5'
 import { createSchedule } from '@xen-orchestra/cron'
+import { createLogger } from '@xen-orchestra/log'
```

Comment:
Keep these lines sorted.

```suggestion
import { createLogger } from '@xen-orchestra/log'
import { createSchedule } from '@xen-orchestra/cron'
```